ui/js/controllers/main.js

```diff
@@ -26,7 +26,7 @@ treeherderApp.controller('MainCtrl', [
             return "[" + ufc + "] " + $rootScope.repoName;
         };
 
-        $scope.closeJob = function() {
+        $rootScope.closeJob = function() {
```

Comment:
Just noting for posterity we invoke `closeJob()` in the plugin controller [here](https://github.com/mozilla/treeherder/blob/89400acafb80b744b4f3b60b1d8e8ff84f6fb3f2/ui/plugins/pluginpanel.html#L275) but I assume this conversion to $rootScope is ok for that?